app/waf/src/services/license.service.ts

```diff
@@ -133,7 +137,16 @@ export class LicenseManager {
 
   // // BIG-IQ ✓<-->x BIG-IP
   // private async licViaBIQ() { }
-  // private async unLicViaBIQ() { }
+  private async unLicViaBIQ(adc: Adc): Promise<void> {
+    let address = adc.management.connection!.ipAddress;
+    let mac = '';
+    for (let net of Object.keys(adc.networks)) {
+      if (adc.networks[net].type === 'mgmt') {
+        mac = adc.management.networks[net].macAddr!;
```

Comment:
Be careful that `adc.networks` exists but `adc.management.networks` doesn't. 
the reclaimFuncs.license in `deleteOn` has no `try..catch..`. When VE is powered partially, is it possible to make reclaiming always failed because of "reading `adc.management.networks` undefined error"?

Reply:
If the VE is in power error state, we should not revoke license, right?